package/patterns-yast.spec

```diff
@@ -51,107 +50,124 @@ Provides:       pattern-visible()
 
 Requires:       libyui-ncurses-pkg
 Requires:       yast2
+Requires:       yast2-firewall
+Requires:       yast2-packager
+Requires:       yast2-theme
+
+Recommends:     yast2-metapackage-handler
+Recommends:     yast2-support
+
+Suggests:       yast2-firstboot
+Suggests:       yast2-snapper
+
+# SLE only
+%if !0%{?is_opensuse}
+Requires:       yast2-registration
+# yast2 clone_system is expected to be installed by default (sle-beta)
+Recommends:     autoyast2-installation
+%endif
+
 %if 0%{?is_opensuse}
-# opensuse only, see bsc#1125040
 Requires:       yast2-alternatives
+Recommends:     yast2-vm
+%endif
+
+# Don't install update facilities on Tumbleweed
+%if 0%{?sle_version}
+Requires:       yast2-online-update
+Requires:       yast2-online-update-frontend
+Requires:       yast2-update
+Suggests:       yast2-online-update-configuration
 %endif
+
+%description yast2_basis
+YaST tools for basic system administration.
+
+%files yast2_basis
+%dir %{_docdir}/patterns
+%{_docdir}/patterns/yast2_basis.txt
+
+################################################################################
+
+%package yast2_desktop
+%pattern_basetechnologies
+Summary:        YaST Desktop Utilities
+Group:          Metapackages
+Provides:       pattern() = yast2_desktop
+Provides:       pattern-icon() = yast
+Provides:       pattern-order() = 1222
+Provides:       pattern-visible()
+
 Requires:       yast2-country
-Requires:       yast2-firewall
-Requires:       yast2-hardware-detection
+Requires:       yast2-installation
+Requires:       yast2-services-manager
+Requires:       yast2-sysconfig
+Requires:       yast2-users
+
+Recommends:     yast2-fonts
+Recommends:     yast2-journal
```

Comment:
in fact I think that journal is useful for both server and client as it logs there from both kind of tools.

Reply:
ah, it is there. so nothing to do.